.github/workflows/build.yml

```diff
@@ -228,6 +229,7 @@ jobs:
     - name: Copy premake files
       run: |
         cp -r premake/* .
+        cp -r resource/* .
```

Comment:
https://stackoverflow.com/questions/1997172/is-there-a-linux-equivalent-of-windows-resource-files

I don't think Linux need resource files